src/game_objects/grid.h

```diff
@@ -49,7 +49,8 @@ class Grid : public GameObject{
     bool verify_build(ConstructType type, int col, int row);
 
     // Builds a construct at a location.
-    Construct* build(ConstructType type, int col, int row, bool graphical, int id = -1);
+    Construct* build(ConstructType type, int col, int row, bool graphical, 
+        float fx, float fy, float fz, int id = -1);
```

Comment:
Same as above, don't need graphical bool anymore. Also, id should be set to 0 by default, since that's the cue that it is on the server.